Automattic/wp-calypso
# Help Center: Doc title is missing when minimized

Fixes https://github.com/Automattic/dotcom-forge/issues/8127

## Proposed Changes

Add the title of the article to the params that we pass when we open the docs in the Help Center from `/home`

## Why are these changes being made?


This can only be reproduced when the Help Center is opened directly to an article. The header relies on the article title to be in a query param which is missing when linked to from outside the Help Center

<img width="577" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/1bcd6b2f-bf72-4c31-9da6-9828abdd6af2">


## Testing Instructions



* Go to `/home`
* Click on an article in the Get help panel, it should open in the Help Center
* Minimize the Help Center, you should be able to see the title
